python/ppml/example/pytorch_nn_lr/pytorch_nn_lr_1.py

```diff
@@ -50,8 +54,10 @@ def forward(self, x: List[Tensor]):
 
 @click.command()
 @click.option('--load_model', default=False)
-def run_client(load_model):
-    df_train = pd.read_csv('./data/diabetes-vfl-1.csv')    
+@click.option('--data_path', default="./data/diabetes-vfl-1.csv")
+def run_client(load_model, data_path):
+    init_fl_context('1')
```

Comment:
should `id` will be an integer instead?

Reply:
We can do this. And user must pass an integer when calling `init_fl_context` and the client ID could only be set in this method.

All the protobuf types of ID are String now, I would open another PR to modify all of them.